src/ctap/response.rs

```diff
@@ -135,7 +136,12 @@ pub struct AuthenticatorGetInfoResponse {
     pub firmware_version: Option<u64>,
     pub max_cred_blob_length: Option<u64>,
     pub max_rp_ids_for_set_min_pin_length: Option<u64>,
+    // Missing response fields as they are only relevant for internal UV:
```

Comment:
nit: I found the "internal" here a bit confusing even though that's the term used in FIDO Registry. But I can't think about an ideal adjective. Maybe "built-in" would more reflect that UV is fully executed within the authenticator boundaries?

Reply:
I added an explanation below with instructions and examples.

I already use "internal UV" in multiple places already. Generally, I expect a reader to consult the specification.